test/19.js

```diff
@@ -0,0 +1,17 @@
+const expect = require('chai').expect;
+let solution = require('../solutions/19').solution;
+
+describe('Pairs sum to x', () => {
+  it('should return pairs of number that sum to x', () => {
```

Comment:
The description of each test case should be different, so that if one of them fails we know which one right away from reading the console.  For example, ` should return pairs of number from [1,2,3,4,5] that sum to 6`